src/Tasks/ManifestUtil/ApplicationManifest.cs

```diff
@@ -255,13 +262,12 @@ public string OSSupportUrl
         }
 
         /// <summary>
-        /// Specifies the minimum required OS version required by the application.
+        /// Gets or sets the minimum OS version required by the application.
+        /// </summary>
+        /// <remarks>
         /// An example value is "5.1.2600.0" for Windows XP.
-        /// If this input is not specified a default value is used.
```

Comment:
Looking at the source code, I don't see that a default value is used, so I removed this. Also, Windows XP is now the minimum supported OS for .NET Framework (4).

Reply:
Line 261 here says the default value / minimum supported OS is a version of Windows 98SE. Should that line also be deleted?

```
 /// The default value is the minimum supported OS of the .NET Framework, which is "4.10.0.0" for Windows 98SE.
```

Reply:
Yes, it is deleted.

Reply:
[This](https://github.com/dotnet/msbuild/pull/6284/files#diff-c990762d052397b3b84d63ab61075155c0458affcddcb0640afc5e1ec578407fL226-L229) looks like setting a default value to me.

Reply:
Ah, thanks. I will update the comments then.

```
            if (String.IsNullOrEmpty(OSVersion))
            {
                OSVersion = !WinXPRequired ? Constants.OSVersion_Win9X : Constants.OSVersion_WinXP;
            }
```